src/Runner.Sdk/Util/StringUtil.cs

```diff
@@ -127,5 +127,23 @@ public static string SubstringPrefix(string value, int count)
         {
             return value?.Substring(0, Math.Min(value.Length, count));
         }
+
+        public static string TimespanToHumanReadableString(TimeSpan duration)
+        {
+            if (duration.Duration().TotalSeconds == 0) {
```

Comment:
nit: I like the idea of this function, but I'm really not sure how often we will use it and really for this pr its only being used to say `last 10 minutes`, should we consider a constant here instead and revisit this if there are more places we want to use it?

Reply:
Thanks for the feedback! I can understand that and was kinda thinking the same as I was writing it. I opted to include it as it's testable and gives a much cleaner user experience since you can pass any `TimeSpan` to this, will automatically handle different intervals, so the warning message doesn't just have to say `last x minutes` and can break up into `last 2 hours, 40 minutes` or whatever. 

I can totally understand the rational though and it's easy enough to add back in if we decide to use something like that in the future so Ill get rid of it for now!

Reply:
Got rid of the function and the helper function in favour or a more simple timespan. Still need to sync with product about the wording